scripts/screens/MedDenScreen.py

```diff
@@ -1,9 +1,11 @@
+from random import choice
+
 import i18n
 import pygame
 import pygame_gui
 
 from scripts.cat.cats import Cat
-from scripts.clan import HERBS
+from scripts.clan_resources.herb.herb_supply import MESSAGES
```

Comment:
Don't import that there, import it here!

Also, it'll need something like the following to protect it:

```
currently_loaded_lang = None
```
and
```
if currently_loaded_lang != i18n.config.get("locale"):
    MESSAGES = load_string_resource("screens/med_den_messages.json")
    currently_loaded_lang = i18n.config.get("locale")
```

with the latter appearing any time we're about to call `MESSAGES` - only once in the function tho, worry not.

Reply:
We also need it in the herb_supply class too though, I was under the impression it was better to load a resource in one place and then import it where needed, is that not the case?

Reply:
I could've sworn we don't use it in herb_supply, disregard me.

Reply:
It's being used in `get_status_message`